### DISCUSSION TITLE
Standalone prompts options 1-4 for which model to use, every time - and downloads model again!

### DISCUSSION THREAD

**Discussion by ewebgh33** [erew123/alltalk_tts#369] [Q&A]
Hi
I installed standalone and everything works (tts, rvc, etc).

But if I shut it down (with control-C as it's suggested to do) and go to re-open it another time,
then on start it asks, every time, what model to use.
I select 3) xtts
And it re-downloads the models, even though I've done this like 5 times now.

What setting is not being saved, and where, that it keeps asking me and re-downloading the models when they're already on the drive?
xtts is a 2gb model and this is painful.

OR 
Is it simply asking which models I want to use, and I should be selecting option 4) " I have my own models already"
Because I kind of don't want to have to choose that EVERY time, you know?
And the fact that this says "avaialble FIRST TIME start up models" makes me think, it's actually just not remembering that I've run AlLTalk a bunch of times already. If this isn't first time start up why is it showing me the "available first time start up" models selection?

Thanks

**Comment by erew123** [comment]
Hi @ewebgh33 Please see firstrun_model https://github.com/erew123/alltalk_tts/wiki/Configuration-File:-confignew.json

That **should** always get set to false when that portion of the script is run, bar the file being read only or no permissions access. I cannot think of another reason why it wouldnt work.

You can set that to `false` or always skip the downloads in the menu choices with option 4.

Thanks
